so-eclipse-plugin/src/com/sohelper/ui/QuestionPage.java

```diff
@@ -77,9 +78,15 @@ public void widgetSelected(SelectionEvent event) {
 				
 		Button votedUpCheckbox = new Button(optionsCont, SWT.CHECK);
 		Label onlyVotedUpLbl = new Label(optionsCont, SWT.NONE);
-		onlyVotedUpLbl.setText("Voted only (soon)");
-		votedUpCheckbox.setSelection(false); 
-		votedUpCheckbox.setEnabled(false);
+		onlyVotedUpLbl.setText("Voted only");
+		
+		votedUpCheckbox.addSelectionListener(new SelectionAdapter() {
+			@Override
```

Comment:
Just a little note: Mixing tabs and spaces is bad, specially when viewing code on repositories. It would be great if you change all spaces here to tab(s).

You can also use lambda expression (since we're using Java8):

`votedUpCheckbox.addSelectionListener(() -> {...});`